var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -22,6 +22,9 @@ class Hdf5(AutotoolsPackage):
 
     test_requires_compiler = True
 
+    # See https://github.com/spack/spack/pull/21900
+    install_libtool_archives = False
```

Comment:
Isn't this already inherited from the base class?

Reply:
That's true but now we rely on the `*.la` files to be removed (we don't try to make sure that they are correct). So, it's a precaution against someone blindly changing the value to `True`, either here or in the baseclass.

Reply:
Can you add this explanation as a comment above this line? I see why something like this can be good for a package like hdf5, but it's not obvious by just looking at the code (to me at least 😊 )

Reply:
There is already a comment with a link to this discussion :) But fine, I'll update it.

Reply:
Thanks for updating the comment, much clearer now :wink: